dice_ml/model_interfaces/base_model.py

```diff
@@ -58,4 +61,6 @@ def get_num_output_nodes(self, inp_size):
         return self.get_output(temp_input).shape[1]
 
     def get_num_output_nodes2(self, input):
```

Comment:
Should we rename this function to something else?

Reply:
Since this is a public function in a public module and already shipping in DiCE ML, it may not be easy to rename this function. 

The best way might be to send a deprecation message for this method saying that this would be deprecated and renamed to <blah> and make a DiCE ML release and then remove it in the next release.